content/en/docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -273,9 +273,11 @@ which matches the behavior when this feature is disabled.
 
 ## Taint Nodes by Condition
 
-Version 1.8 introduces an alpha feature that causes the node controller to create taints corresponding to
-Node conditions. When this feature is enabled (you can do this by including `TaintNodesByCondition=true` in the `--feature-gates` command line flag to the scheduler, such as
-`--feature-gates=FooBar=true,TaintNodesByCondition=true`), the scheduler does not check Node conditions; instead the scheduler checks taints. This assures that Node conditions don't affect what's scheduled onto the Node. The user can choose to ignore some of the Node's problems (represented as Node conditions) by adding appropriate Pod tolerations.
+In version 1.12, `TaintNodesByCondition` feature is promoted to beta, so node lifecycle controller automatically creates taints corresponding to
+Node conditions.
+> this feature can be disabled by including `TaintNodesByCondition=false` in the `--feature-gates` command line flag to the scheduler, such as `--feature-gates=...,TaintNodesByCondition=false`).
```

Comment:
Why quoted text?
Also the ) at the end of the sentence should be removed.

Reply:
Also "this" should be capitalized.

Reply:
@bsalamat Quoted is for just explanation purpose - to tell user it's a beta feature, and they can disable it if they want. If it's unnecessary, it can be totally removed.

Reply:
The quoted text is rendered with lighter font color and gives the impression that it is not important. I would put the text under a sub-section called "how to disable" or something like that.

Reply:
To avoid confusion, I would like to remove the section. Even if people needs to disable it, they should be able to infer which parameter to pass.
And I notice that most of beta features didn't mention how to disable them.